### ISSUE TITLE
Adding TS plugins outside of working directory

### ISSUE THREAD

**Issue by geelen** [fab-spec/fab#311]
Following on from #306, and https://github.com/fab-spec/plugin-add-fab-id/pull/2, I've discovered that the `@rollup/plugin-typescript` is failing to activate on paths outside of the root directory of the project. Weirdly, when running `yarn link @fab/actions` to test things locally, it doesn't fail, nor can I reproduce the exact failure in #306 itself.

Basically, something strange in the neighbourhood.

In investigating, I've found that `@rollup/plugin-typescript` has moved on several versions and now does real typechecking, whereas we are using an old version that just strips the types and calling `tsc` in a separate thread. So I think the best solution would be to move to [`@rollup/plugin-sucrase`](https://github.com/rollup/plugins/tree/master/packages/sucrase) which has the type-annotation-stripping behaviour we want.

TL;DR TODO: switch to sucrase in the rollup
